code/modules/mob/living/say.dm

```diff
@@ -30,6 +30,7 @@ var/list/department_radio_keys = list(
 	  ":U" = "Supply",		"#U" = "Supply",		".U" = "Supply",
 	  ":Z" = "Service",		"#Z" = "Service",		".Z" = "Service",
 	  ":P" = "AI Private",	"#P" = "AI Private",	".P" = "AI Private",
+	  ":!" = "Response Team", "#!" = "Response Team", ".!" = "Response Team",

```

Comment:
I thought ! was the 'emote' bind.
IE: say ";! pops" produces "CharacterName pops" on radio.

I fear this is another example of a bind that is already reserved for something else.

Reply:
Wait what? That's a thing you can do? Who would have thought finding an unused symbol would be this hard.

Reply:
Yes, that was my sentiment when I added new channels. Eventually I resorted to using weird symbols like - and _ out of desperation.

Reply:
What about '?' then? For strange channel?

Reply:
Also, someone explain why there are three things you can type ( :  #  and  . ) for radio. Which does which?

Reply:
I am fairly sure I tried ? before and it wasn't a good idea.

: # and . all behave identically. There is no good reason to have all three. We should really just use . or # 
The : is too close to ; and this regularly results in people typing ;h instead of :h

Reply:
I see it as : being a specific channel while ; looking like a common headset (with the , being the mic arm). It's the others I would have removed